### PR TITLE
Font fix for WPF

### PR DESCRIPTION
**Pull request by GantMan**
custom fontFamily props were not working in WPF because assets are accessed differently in WPF.  This PR re-arranges the string to be consumed by a URI that can properly access and set the `fontFamily` for text.

image

### CODE REVIEW

**Comment by matthargett** [comment]
Can this be made more congruent (and/or maybe shared) with the loading of embedded resources in InitialtizeAsync() here: https://github.com/ReactWindows/react-native-windows/blob/fb60d23ea1d6434e2e394ddbd33816f65ca9b33e/ReactWindows/ReactNative.Shared/Bridge/JavaScriptBundleLoader.cs#L98 
?

File: ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs [hunk 1]
```diff
@@ -232,11 +233,25 @@ private void UpdateTextBlockCore(TextBlock textBlock, bool measureOnly)
...
 
+            if (_fontFamily != null)
+            {
+                // convert font string into something WPF can use
+                // e.g. FontFamily(new System.Uri("pack://application:,,,/"), "./Assets/#fontname")
+                string[] path = _fontFamily.Split('/');
+                path = path.Take(path.Count() - 1).ToArray();
+                string cleanPath = "./" + string.Join("/", path) + "/";
+                string[] fontParts = _fontFamily.Split('#');
+                textBlock.FontFamily = new FontFamily(new System.Uri("pack://application:,,,/"), cleanPath + "#" + fontParts.Last());
```
Comment (rozele) [inline ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs]: >application [](start = 77, length = 11)

Do you have a pointer to some documentation on this URI?

**Comment by GantMan** [comment]
Scroll to "Referencing Font Resource Items from Code"
https://msdn.microsoft.com/en-us/library/ms753303(v=vs.110).aspx

There's also a corresponding stack overflow answer from similar questions.

100% transparency, I have no clue what secret knock this URI is doing.  It's pretty crazy with the "tres comas" - and additional pomp.  But it works.

My goal was to take any UWP friendly code and make it work for WPF

**Comment by GantMan** [comment]
In addition - Leon will be using this to scan for any other asset related issues.  Once we have a large list of asset problems, a more centralized solution might reveal itself.  

Regardless, the `fontFamily` string needs to parsed differently for WPF.

**Comment by rozele** [comment]
Could we put the MSDN reference in a comment in the code?

---
In reply to: [270129550](https://github.com/ReactWindows/react-native-windows/pull/944#issuecomment-270129550) [](ancestors = 270129550)

**Comment by GantMan** [comment]
@rozele - comment added.

**Pull request merged**